cmd/limactl/info.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/lima-vm/lima/pkg/limayaml"
+	"github.com/lima-vm/lima/pkg/version"
+	"github.com/spf13/cobra"
+)
+
+func newInfoCommand() *cobra.Command {
+	infoCommand := &cobra.Command{
+		Use:   "info",
+		Short: "Show diagnosis information",
```

Comment:
I'm not a native speaker, but I think "diagnostic information" would be a better term. "Diagnosis" is a noun (it is a "diagnostic procedure"), and using it as an adjective feels wrong to me.

Reply:
Thanks, fixed